src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -491,4 +497,27 @@ public static String replaceTokens(Map<String, ?> tokens, String value) {
 
         return buf.toString();
     }
+
+    public static void setupIDEResourceCopy(@Nonnull final Project project) {
+        if (project.getPlugins().hasPlugin(IdeaPlugin.class)) {
+            final IdeaPlugin idea = project.getPlugins().getPlugin(IdeaPlugin.class);
+            project.getTasks().create("copyIdeaResources", CopyIDEAResources.class, task -> task.configure(idea.getModel()));
+        } else {
+            project.getLogger().warn("'idea' plugin is not found! IDEA resources will not be copied.");
+            project.getTasks().create("copyIdeaResources", Task.class); // Create dummy class
+        }
+
+        if (project.getPlugins().hasPlugin(EclipsePlugin.class)) {
+            project.getTasks().create("copyEclipseResources", CopyEclipseResources.class, task -> {
+                task.dependsOn("eclipse");
+                project.getTasks().named("eclipse").configure($ -> $.doLast($$ -> {
+                    final EclipseModel eclipse = project.getExtensions().findByType(EclipseModel.class);
+                    task.configure(eclipse, project);
+                }));
+            });
```

Comment:
Why does this task need to depend on `eclipse`?

Reply:
Because the model and / or the information I need from it is `null` when the eclipse task hasn't been run.